src/frontend/flight/handler.rs

```diff
@@ -162,6 +163,10 @@ impl SeafowlFlightHandler {
             });
         }
 
+        if cmd.format != TableFormat::Delta as i32 {
+            return Err(SyncError::NotImplemented);
```

Comment:
Maybe we can just use `SyncError::InvalidMessage` here since it's going to be temporary anyway.

Reply:
I'm inclined to keep it. This functionality being unimplemented is temporary, but having some functionality unimplemented is permanent :)

"Not implemented" exists as a distinct concept in some languages e.g. Python: https://docs.python.org/3/library/constants.html#NotImplemented

Reply:
Fair point, we may need it in future.